pkg/login/login.go

```diff
@@ -106,11 +106,14 @@ func createSecret(ctx context.Context, c client.Client, app *apiv1.App, secretNa
 	data := map[string][]byte{}
 	promptOrder, _ := app.Status.AppSpec.Secrets[secretName].Params.GetData()["promptOrder"].([]string)
 	for _, key := range promptOrder {
-		if _, ok := app.Status.AppSpec.Secrets[secretName].Data[key]; ok {
-			value, err := prompt.Password(key)
+		if def, ok := app.Status.AppSpec.Secrets[secretName].Data[key]; ok {
+			value, err := prompt.Password(fmt.Sprintf("%s (default: %s)", key, def))
 			if err != nil {
 				return err
 			}
+			if len(value) == 0 {
```

Comment:
Is it worth it to be a `strings.TrimSpace` on `value` here (and below)?

Reply:
Imo, no. If the user enters spaces, I think those should be used as the value instead of changing it to the default.